### DISCUSSION TITLE
rowGroupMode both subheader and rowspan together

### DISCUSSION THREAD

**Discussion by RayhanBryan** [primefaces/community#548] [PrimeVue] [answered]
How can I make datatable with rowGroupMode both subheader and rowspan together in PrimeVue?

**Comment by tugcekucukoglu** [comment, marked as answer]
'rowgroup' mode uses `rowGroupMode="subheader"` and 'rowspan' mode uses `rowGroupMode="rowspan"`. It would be beneficial to choose one of them.
